style-guide/code-style/html/README.md

```diff
@@ -0,0 +1,6 @@
+# HTML 
+
+## Formatting
+* Don't use inline styles, unless you are generating dynamic styles in your templating system.
```

Comment:
Some Inlining of styles is recommended or advocated for performance. e.g - displaying above the fold content. Check [here](https://developers.google.com/speed/docs/insights/OptimizeCSSDelivery) and let me know what you think ? Can maybe add some reference or content from this ?